frontend/src/main/components/Alias/AliasTable.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import OurTable from "main/components/OurTable";
+import { toast } from "react-toastify";
+import { useBackendMutation } from "main/utils/useBackend";
+
+export default function AliasTable({ alias }) {
+  const testid = "AliasTable";
+
+  const objectToAxiosParamsApprove = (user) => ({
+    // Stryker disable next-line all
+    url: `/api/currentUser/updateAliasModeration`,
+    method: "PUT",
+
+    /* Stryker disable all */ //not tested because it is mocked
+    params: {
+      id: user.id,
+      approved: true,
+    },
+    /* Stryker restore all */
+  });
+
+  const approveMutation = useBackendMutation(objectToAxiosParamsApprove, {
+    onSuccess: (user, propAlias) => {
+      toast(`Alias ${propAlias.proposedAlias} for id ${user.id} approved!`);
+    },
+    // Stryker disable next-line all
+    onError: (err) => {
+      // Stryker disable next-line all
+      toast.error(`Error approving alias: ${err.message}`);
+    },
+  });
+
+  const objectToAxiosParamsReject = (user) => ({
+    // Stryker disable next-line all
+    url: `/api/currentUser/updateAliasModeration`,
+    method: "PUT",
+
+    /* Stryker disable all */ //not tested because it is mocked
+    params: {
+      id: user.id,
+      approved: false,
+    },
+    /* Stryker restore all */
+  });
+
+  const rejectMutation = useBackendMutation(objectToAxiosParamsReject, {
+    onSuccess: (user, propAlias) => {
+      toast(`Alias ${propAlias.proposedAlias} for id ${user.id} rejected!`);
+    },
+
+    /* Stryker disable all */
+    onError: (err) => {
+      toast.error(`Error rejecting alias: ${err.message}`);
+    },
+    /* Stryker restore all */
+  });
```

Comment:
This should be tested.  Please remove the stryker exception and add a test.  We have many examples of how to test for toast messages; it isn't hard.